packages/runtime/container-runtime/src/deltaScheduler.ts

```diff
@@ -86,7 +90,18 @@ export class DeltaScheduler {
 
                 // eslint-disable-next-line @typescript-eslint/no-floating-promises
                 this.deltaManager.inbound.pause();
+
                 setTimeout(() => {
+                    if (this.schedulingLog) {
+                        this.logger.sendTelemetryEvent({
+                            eventName: "DeltaManagerPaused",
+                            duration: elapsedTime,
+                            numberOfOps: this.deltaManager.inbound.length,
+                            numberofOpsProcessed: this.queueLengthBeforeBatchBegin - this.deltaManager.inbound.length,
```

Comment:
Similar as below - numberofOpsProcessed is not what it says, as inbound can be receiving ops as we go. It would be great to take a look at how we can capture actual number of ops processed.

I see couple ways:
- Add extra payload on "batchEnd" event that signifies number of ops processed
- batchEnd already has message payload. If we add message payload on beginBatch, we can capture sequence numbers and substract, getting number of ops processed
- we can listen on "op" method from container runtime and count (simpler, but a bit more expensive probably from CPU processing).

Reply:
I tried listening to ops. Let me know what you think.